src/stylus/elements/_code.styl

```diff
@@ -1,4 +1,4 @@
-code
+code, kbd
   background: $grey.base
```

Comment:
make a separate rule for `background` and `color` for `code` instead of sharing them for `code, kbd` and then overwriting for `kbd`